.github/label-actions.yml

```diff
@@ -1,4 +1,4 @@
-'reproduction:needed':
+'auto:reproduction':
```

Comment:
You should search for the _old_ labels to find any files that still use those.

This file needs updating:

https://github.com/renovatebot/renovate/blob/chore/label-actions-more/docs/development/triage-guide.md

Reply:
This comment will be resolved by PR:

- #24669